ports/libbson/vcpkg.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "libbson",
+  "version": "1.17.6",
```

```suggestion
  "version-semver": "1.17.6",
```
This can be served by [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver).

---

I don't believe mongo says that these follow semver.

---

Yeah, we should not do `version-semver` unless the upstream says explicitly that they're semver

---

Thanks for your help to review @BillyONeal @strega-nil-ms.